monitoring/rid_qualifier/test_executor.py

```diff
@@ -0,0 +1,29 @@
+from monitoring.rid_qualifier.aircraft_state_replayer import TestHarness, TestBuilder
+import asyncio
+import arrow
+from monitoring.rid_qualifier.utils import RIDQualifierTestConfiguration, RIDQualifierUSSConfig
+
+def build_test_configuration(locale: str, auth_spec:str, injection_base_url:str, injection_suffix:str=None, allocated_track = 0) -> RIDQualifierTestConfiguration: 
```

Comment:
The three parameters `injection_base_url`, `injection_suffix`, and `allocated_track` are only used to construct a `RIDQualifierUSSConfig` -- it would be simpler/clearer to just pass a single pre-instantiated `RIDQualifierUSSConfig` and have the caller construct it.

Reply:
OK I have now changed this.